chainer/link.py

```diff
@@ -578,6 +578,26 @@ def serialize(self, serializer):
         for name in self._persistent:
             d[name] = serializer(name, d[name])
 
+    def count_params(self):
+        """Count the total size of parameters.
+
+        If the link containts uninitialized parameters, this raises warnings.
```

Comment:
this -> this method
warnings -> a warning

Reply:
Thanks. fixed.